tests/test_minhash.py

```diff
@@ -1872,6 +1878,21 @@ def test_merge_scaled():
     for k in mh2.hashes:
         assert k in mh3.hashes
 
+def test_add_is_symmetric():
```

Comment:
whups, just caught this - these two tests (`test_add_is_symmetric` and `test_or_equals_add`  are both working with empty MinHashes, so none of them are actually testing anything, since an empty MinHash is always equal to  an empty MinHash :).

So, a few requests!

* use `mh.add_hash(...)` to add numbers in
* add an `assert mh3` to make sure that it's got something in it